docs/task-cluster.md

```diff
@@ -0,0 +1,81 @@
+# TaskCluster
```

Comment:
Nit: `Taskcluster` is the correct capitalization.

Reply:
good to know, fixed everywhere